local-volume/provisioner/deployment/docker/Dockerfile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM centos:7
+FROM k8s.gcr.io/debian-base-amd64:0.4.0
 
-RUN yum update -y && \
-    yum install -y e2fsprogs && \
-    yum install -y hdparm && \
```

Comment:
cc @dhirajh do you remember why hdparm was needed?

Reply:
@msau42 hdparm was included in anticipation that someone might want to use ATA secure erase https://ata.wiki.kernel.org/index.php/ATA_Secure_Erase
as an option. However, I didn't have a script for it out of the box.

Reply:
thanks, then I think it is fine to remove in this case. We can add it back if we need it.